Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/mysql/MySQLNode.java

```diff
@@ -76,8 +95,9 @@ private BasicActionBuilder initActionBuilder(Runnable runnable) {
 
     @Override
     public List<NodeAction> getNodeActions() {
-        boolean updating = SERVER_UPDATING.equals(serverState);
-        boolean running = ServerState.READY.equals(serverState);
+        boolean running = StringUtils.equalsIgnoreCase("READY", serverState);
+        boolean updating = StringUtils.equalsIgnoreCase(SERVER_UPDATING, serverState);
```

Comment:
make status such as "READY" as a variable ?

Reply:
should be laid in later refactor , not track2 refactor